CCSA_ChatApp.Infrastructure/Services/IMessageHistoryService.cs

```diff
@@ -0,0 +1,16 @@
+﻿using CCSA_ChatApp.Domain.DTOs.MessageDTOs;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CCSA_ChatApp.Infrastructure.Services
+{
+    public interface IMessageHistoryService
+    {
+        List<MessageDTO> FetchMessagesBySenderId(Guid senderId);
```

Comment:
IEnumerable for List... Buh it's fine to me though

Reply:
Pls use Ienumerable

Reply:
I guess its only a problem for fluent nhibernate, I'm not sure though

Reply:
Okay I'll change it